### PR TITLE
Adds zavod alt clothes for warden

### PR DESCRIPTION
**Pull request by LoreCoder**
Adds in an alt version of the Zavodskoi warden coat and hat like with most Zavod things, as well as making it actually selectable via the loadout.

image

### CODE REVIEW

**Comment by LoreCoder** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -110,7 +110,9 @@
 /datum/gear/head/warden/New()
 	..()
 	var/list/wardenhead = list()
-	wardenhead["warden hat, zavod"] = /obj/item/clothing/head/warden
+	wardenhead["warden hat"] = /obj/item/clothing/head/warden
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:113]: This is a parent item, not supposed to be used.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
I hope you followed the palette by using the eyedropper tool.

**Comment by alsoandanswer** [comment]
Looks like you're pulling from your master. You forgot to make a branch.

**Comment by LoreCoder** [comment]
Closing PR to make a branch real quick and remove the original warden hat as requested.

**Pull request closed without merging**